example/use_cases/apartment_application/apartment_application.go

```diff
@@ -0,0 +1,150 @@
+// This is a full example flow of an apartment verifying the age of a potential tenant.
+
+// The apartment will create a Presentation Request that is to be fulfilled by the tenant.
+// The tenant will fulfil the Presentation Request by submitting a Presentation Submission.
+// This presentation submission will contain a verifiable credential that has been previously issued and signed from the government issuer.
+
+// The tenant will verify that the apartment's presentation request is valid and the apartment will also verify that the tenant's
+// presentation submission is valid.
+
+// At the end the apartment will verify the authenticity of the presentation submission and will be able to verify the birthdate of the tenant.
+
+package main
+
+import (
+	"crypto/ed25519"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/goccy/go-json"
+	"github.com/stretchr/testify/assert"
+)
+
+func main() {
+
+	/**
+		 Step 1: Create new entities as DIDs. Govt Issuer, User Holder, and Apartment Verifier
+	**/
+
+	// User Holder
+	holderDIDPrivateKey, holderDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
+	holderJWK, _ := cryptosuite.JSONWebKey2020FromEd25519(holderDIDPrivateKey.(ed25519.PrivateKey))
+	holderSigner, _ := cryptosuite.NewJSONWebKeySigner(holderJWK.ID, holderJWK.PrivateKeyJWK, cryptosuite.Authentication)
+	holderVerifier, _ := cryptosuite.NewJSONWebKeyVerifier(holderJWK.ID, holderJWK.PublicKeyJWK)
+
+	// Apt Verifier
+	aptDidPrivateKey, aptDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
+	aptJWK, _ := cryptosuite.JSONWebKey2020FromEd25519(aptDidPrivateKey.(ed25519.PrivateKey))
+	aptSigner, _ := cryptosuite.NewJSONWebKeySigner(aptJWK.ID, aptJWK.PrivateKeyJWK, cryptosuite.Authentication)
+	aptVerifier, _ := cryptosuite.NewJSONWebKeyVerifier(aptJWK.ID, aptJWK.PublicKeyJWK)
+
+	// Government Issuer
+	govtDidPrivateKey, govtDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
+	govtJWK, _ := cryptosuite.JSONWebKey2020FromEd25519(govtDidPrivateKey.(ed25519.PrivateKey))
+	govtSigner, _ := cryptosuite.NewJSONWebKeySigner(govtJWK.ID, govtJWK.PrivateKeyJWK, cryptosuite.Authentication)
+
+	fmt.Print("\n\nStep 1: Create new DIDs for entities\n\n")
+	fmt.Printf("Tenant: %s\n", string(*holderDIDKey))
+	fmt.Printf("Apartment: %s\n", string(*aptDIDKey))
+	fmt.Printf("Government: %s\n", string(*govtDIDKey))
+
+	/**
+		 Step 2: Government issuer issues credentials to holder claiming age. The government issuer then signs the verifiable credentials to holder claiming age
```

Comment:
nit: `issuer issues a credential to the holder providing their age. The government...`